.github/test-infra/eks/zarf.yaml

```diff
@@ -0,0 +1,45 @@
+kind: ZarfPackageConfig
+metadata:
+  name: distro-eks
+  description: "Deploy an EKS K8s cluster"
+  architecture: multi
+  version: "0.15.1"
+
+variables:
+  - name: CLUSTER_NAME
+  - name: PERMISSIONS_BOUNDARY_ARN
+  - name: PERMISSIONS_BOUNDARY_NAME
+  - name: AMI_ID
+    default: ami-068ab6ac1cec494e0
+
+components:
+  - name: load-eksctl
+    required: true
+    actions:
+      onDeploy:
+        after:
+          # Remove existing eksctl
+          - cmd: rm -f eksctl
```

Comment:
This also probably shouldnt be a zarf package (see other comment). 

Separately, i could question if it makes sense to just use the basic eksctl deployment instead of a config that is more representative of an EKS deployment that would be done in a prod env (for example like some of deliveries IaC). This would be beneficial for actually catching issues that may not come up in such a basic test. Additionally, it would bring the testing mroe in line functionally with any rke2 testing.

Reply:
Generally agree with sentiment on different eks deployment, but think this is acceptable for first pass - we should still be able to catch something like "irsa no worky" with this.